nipype/interfaces/utility/base.py

```diff
@@ -128,24 +128,36 @@ class Merge(IOBase):
     def __init__(self, numinputs=0, **inputs):
         super(Merge, self).__init__(**inputs)
         self._numinputs = numinputs
-        add_traits(self.inputs, ['in%d' % (i + 1) for i in range(numinputs)])
+        if numinputs > 0:
+            input_names = ['in%d' % (i + 1) for i in range(numinputs)]
+        elif numinputs == 0:
+            input_names = ['in_lists']
+        else:
```

Comment:
so the only question here is whether this should be for `Merge(1)` instead of `Merge(0)`